Source/MQTTnet.Benchmarks/MQTTnet.Benchmarks.csproj

```diff
@@ -14,7 +14,7 @@
         <NuGetAuditMode>all</NuGetAuditMode>
         <NuGetAudit>true</NuGetAudit>
         <NuGetAuditLevel>low</NuGetAuditLevel>
-        <AnalysisLevel>latest-Recommended</AnalysisLevel>
+        <!--<AnalysisLevel>latest-Recommended</AnalysisLevel>-->
```

Comment:
try this on .NET 9, if you commented it out because of the 100s of warning messages
<PropertyGroup>
  <AnalysisLevel>latest</AnalysisLevel>
  <AnalysisMode>recommended</AnalysisMode>
  <AnalysisModeStyle>default</AnalysisModeStyle>
</PropertyGroup>

Reply:
That's 100s of error messages! We can unify this issue in #2120.

Reply:
see https://developercommunity.visualstudio.com/t/Code-Rules-that-were-Informational-are-n/10789409#TPIN-N10795393